packages/bpk-component-modal/src/BpkModalV2/BpkModal.tsx

```diff
@@ -70,7 +70,9 @@ const Header = ({
         ].join(' ')}
       >
         <div className={getClassName('bpk-modal__header-title-container')}>
-          <Heading>{title}</Heading>
+          <span className={getClassName('bpk-modal__title')}>
```

Comment:
I wonder if we need this class at all, as looking at the CSS it looks like we would be applying the same typography style as the `Heading` object already is of `label1`, wdyt?

https://github.com/Skyscanner/backpack/blob/main/packages/bpk-component-modal/src/BpkModalV2/BpKModal.module.scss#L171